decidim-core/app/cells/decidim/redesigned_profile/show.erb

```diff
@@ -0,0 +1,29 @@
+<div class="profile__scaffold__top">
+  <div class="container">
+    <div class="profile__scaffold__top-content">
+      <div>
+        <%= render :avatar %>
+      </div>
+      <div>
+        <%= render :details %>
+      </div>
+      <div>
+        <%= cell "decidim/profile_actions", profile_holder, context %>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="profile__scaffold__middle">
+  <div class="container">
+    <ul class="profile__scaffold__middle-content">
+      <%= render :tabs %>
```

Comment:
The tabs still have a vertical scroll on smaller screens:
![Vertical scroll on tabs](https://user-images.githubusercontent.com/864340/187221945-51e487bd-8f8f-4b15-8e53-1b9999d99670.png)

It also applies to 320px although I purposefully picked a narrow screen for the screenshot.

Reply:
Moved to #9670.